pycaching/geocaching.py

```diff
@@ -383,55 +316,111 @@ def search_quick(self, area, *, strict=False, zoom=None):
         #             # can yield more caches (which are not exactly in desired area)
         #             yield cache
 
-    # add some shortcuts ------------------------------------------------------
-
     def search_rect(
         self,
         rect: Rectangle,
-        *,
```

Comment:
My preference is to keep the asterisk there. This way we ensure people pass the "less important" params always kw-based, which gives us flexibility to shuffle them around & possibly add a new one in the middle.

Reply:
Okay, good reason, resolved in https://github.com/tomasbedrich/pycaching/pull/206/commits/f3c20cb99b05f020d91a241936d3c68b9d674fc0